### ISSUE TITLE
gcc: stack smashing detected error when building using gcc and running tests

### ISSUE THREAD

**Issue by mhw0** [mhw0/libethc#26]
Logs:
```bash
ok 426
*** stack smashing detected ***: terminated
Aborted (core dumped)
```
has to be related to`eth_abi_mpint`

**Comment by mhw0** [comment]
It seems like our ubuntu ci does not use gcc, otherwise it would also fail.

**Comment by mhw0** [comment]
Closing the issue. Fixed in fe18e93fb39bede28a9e068feaba122d8f16ee00
